src/sentinel.c

```diff
@@ -4483,7 +4485,7 @@ void sentinelAskPrimaryStateToOtherSentinels(sentinelValkeyInstance *primary, in
         /* Ask */
         ll2string(port, sizeof(port), primary->addr->port);
         retval = redisAsyncCommand(
-            ri->link->cc, sentinelReceiveIsPrimaryDownReply, ri, "%s is-master-down-by-addr %s %s %llu %s",
```

Comment:
the old sentinel does not recognize the new command, but i suppose it is ok.

Reply:
yes, once we update this part, all sentinels should run on the same version.

Reply:
is there a rolling upgrade scenario for sentinels where the user runs mixed versions of sentinels temporarily?

Reply:
Yes, there is. For example, this Helm chart https://github.com/bitnami/charts/tree/main/bitnami/valkey sets up e.g. a k8s StatefulSet with three pods running a sentinel and a valkey container each.  During a RollingUpdate of the StatefulSet, there will be mixed versions of sentinel and valkey temporarily. The sentinels will still need to find a quorum and initiate failovers during this time (if the primary gets updated)

Reply:
make sense. @hwware I think we should leave this command as it is. Rolling upgrades can take time and we need to be resilient to any failures during this transitional period.

Reply:
> make sense. @hwware I think we should leave this command as it is. Rolling upgrades can take time and we need to be resilient to any failures during this transitional period.

Agree with you. Because @gmbnomis provides me a real world case, let us just leave these commands as they are, util I find a solution to solve the problem.

Reply:
@gmbnomis @PingXie I just update the top comment, and describe a test scenario with mixed version sentinel. Please let me if you have more concern.

Reply:
So, sentinel accepts the new command variants, but it still uses the old ones internally? (which is "IS-MASTER-DOWN-BY-ADDR" only, if I get this correctly)

That looks reasonable. We could get rid of the old forms in some later release when we can be sure that all sentinel versions that are supported during an upgrade understand the new form.

Thank you for explicitly testing the mixed sentinel version upgrade scenario. Highly appreciated!

Reply:
> So, sentinel accepts the new command variants, but it still uses the old ones internally? (which is "IS-MASTER-DOWN-BY-ADDR" only, if I get this correctly)
> 
> That looks reasonable. We could get rid of the old forms in some later release when we can be sure that all sentinel versions that are supported during an upgrade understand the new form.
> 
> Thank you for explicitly testing the mixed sentinel version upgrade scenario. Highly appreciated!

I keep the IS-MASTER-DOWN-BY-ADDR only internal, for user, we still encourage them  to use new command format.  Thanks